actions/user_actions.jsx

```diff
@@ -309,7 +309,8 @@ function onThemeSaved(teamId, onSuccess) {
     const toDelete = [];
 
     for (const themePreference of themePreferences) {
-        if (themePreference.name === '' || themePreference.name === teamId) {
+        const name = themePreference.name;
+        if (name === '' || name === teamId) {
```

Comment:
Oh my bad here on last change.  I missed to provide value for `name` property in deleting the theme.